test/cpp/jit/test_gpu.cpp

```diff
@@ -2866,14 +2866,17 @@ Val* gen_jit_operand(std::pair<ValType, DataType> desc) {
   if (desc.first == ValType::TensorView) {
     return makeSymbolicTensor(2, desc.second);
   } else if (desc.first == ValType::Scalar) {
-    if (desc.second == DataType::Float)
```

Comment:
Do we need to cover half here?

Reply:
No, we don't take any Half scalars. A follow up PR completely removes half and float scalars so we always use double.